From 3694c913db79d87b5eac8936e4ad5c3ce48077e1 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Thu, 14 Feb 2013 14:37:17 +0100
Subject: [PATCH 2/2] Remove appendix sections for dpkg-genchanges,
 dpkg-parsechangelog and dpkg-architecture

These sections are minimal and are already covered by either the main
policy sections or the dpkg documentation. Because they are located at
the end of the appendix, they can be removed safely without incurring
in any section renumbering.
---
 policy.sgml | 57 +--------------------------------------------------------
 1 file changed, 1 insertion(+), 56 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 31d7232..2f52f8b 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2153,7 +2153,7 @@ zope.
 	<p>
 	  The architectures we build on and build for are determined
 	  by <prgn>make</prgn> variables using the
-	  utility <qref id="pkg-dpkg-architecture"><prgn>dpkg-architecture</prgn></qref>.
+	  utility <prgn>dpkg-architecture</prgn>.
 	  You can determine the Debian architecture and the GNU style
 	  architecture specification string for the build architecture as
 	  well as for the host architecture.  The build architecture is
@@ -11349,61 +11349,6 @@ END-INFO-DIR-ENTRY
 	    unchanged into the resulting <file>.changes</file> file.
 	  </p>
 	</sect1>
-
-
-	<sect1 id="pkg-dpkg-genchanges">
-	  <heading>
-	    <prgn>dpkg-genchanges</prgn> - generates a <file>.changes</file>
-	    upload control file
-	  </heading>
-
-	  <p>
-	    This program is usually called by package-independent
-	    automatic building scripts such as
-	    <prgn>dpkg-buildpackage</prgn>, but it may also be called
-	    by hand.
-	  </p>
-
-	  <p>
-	    It is usually called in the top level of a built source
-	    tree, and when invoked with no arguments will print out a
-	    straightforward <file>.changes</file> file based on the
-	    information in the source package's changelog and control
-	    file and the binary and source packages which should have
-	    been built.
-	  </p>
-	</sect1>
-
-
-	<sect1 id="pkg-dpkg-parsechangelog">
-          <heading>
-            <prgn>dpkg-parsechangelog</prgn> - produces parsed
-	    representation of a changelog
-	  </heading>
-
-	  <p>
-	    This program is used internally by
-	    <prgn>dpkg-source</prgn> et al.  It may also occasionally
-	    be useful in <file>debian/rules</file> and elsewhere.  It
-	    parses a changelog, <file>debian/changelog</file> by default,
-	    and prints a control-file format representation of the
-	    information in it to standard output.
-	  </p>
-	</sect1>
-
-        <sect1 id="pkg-dpkg-architecture">
-	  <heading>
-	    <prgn>dpkg-architecture</prgn> - information about the build and
-	    host system
-          </heading>
-
-          <p>
-            This program can be used manually, but is also invoked by
-            <tt>dpkg-buildpackage</tt> or <file>debian/rules</file> to set
-            environment or make variables which specify the build and host
-            architecture for the package building process.
-          </p>
-        </sect1>
       </sect>
 
       <sect id="pkg-sourcetree">
-- 
1.8.1.3

